JS/app.js

```diff
@@ -0,0 +1,45 @@
+//Sec 1
+//1. Concatenation is to combine strings. Interpolation is using a variable to represent a value.
+console.log('hello '+'world.');
+const word = "hello";
+console.log(word + " world.");
+//2.Do Not Repeat Yourself. Program runs more efficient and easier to interpret by humans. Loops.
```

Comment:
it's not necessarily more efficient.  it could even be less efficient because it could require more computation.

**the main concept is that you haven't written duplicate logic.**

it may be more readable by humans, but again, the main purpose is that you don't write the same code more than once